Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+        passwordPrompt = UILabel(frame: .zero)
+        passwordPrompt.text = "Password"
+        passwordPrompt.textColor = .white
+        passwordPrompt.font = .preferredFont(forTextStyle: .headline)
+        loginStackView.addArrangedSubview(passwordPrompt)
+
+        passwordTextField = UITextField(frame: .zero)
+        passwordTextField.textColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        passwordTextField.attributedPlaceholder = NSAttributedString(string: "Type your password",
+        attributes: [NSAttributedString.Key.foregroundColor: UIColor(white: 225.0 / 255.0, alpha: 1.0)])
+        passwordTextField.font = .preferredFont(forTextStyle: .body)
+        passwordTextField.isSecureTextEntry = true
+        passwordTextField.autocapitalizationType = .none
+        passwordTextField.autocorrectionType = .no
+        passwordTextField.tintColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        loginStackView.addArrangedSubview(passwordTextField)
+
+        let passwordSeparator = UIView()
+        passwordSeparator.backgroundColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        loginStackView.addArrangedSubview(passwordSeparator)
+        passwordSeparator.snp.makeConstraints { make in
+            make.height.equalTo(1)
+        }
+    }
+
+    override func setUpButton() {
+        loginButton = UIButton()
+        loginButton.layer.borderWidth = 2
+        loginButton.layer.borderColor = UIColor.white.cgColor
+        loginButton.layer.cornerRadius = 30
+        loginButton.setTitle("LOG IN", for: .normal)
+        loginButton.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .heavy)
+        loginButton.titleLabel?.textColor = .white
+        loginButton.addTarget(self, action: #selector(didTapLoginButton), for: .touchUpInside)
+        stackView.addArrangedSubview(loginButton)
+        loginButton.snp.makeConstraints { make in
+            make.width.equalTo(240)
+            make.height.equalTo(60)
+        }
+
+        activityIndicator = NVActivityIndicatorView(frame: .zero, type: .circleStrokeSpin, color: .white)
+        loginButton.addSubview(activityIndicator)
+        activityIndicator.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+            make.width.height.equalTo(22)
+        }
+    }
+
+    @objc func didTapLoginButton() {
+        requestLoginIfPossible()
+    }
+
+    private func setUpSkipButton() {
+        skipButton = UIButton()
+        skipButton.setTitle("SKIP", for: .normal)
+        skipButton.titleLabel?.font = UIFont.systemFont(ofSize: 18, weight: .bold)
+        skipButton.titleLabel?.textColor = .white
+        skipButton.addTarget(self, action: #selector(didTapSkipButton), for: .touchUpInside)
+        self.view.addSubview(skipButton)
+        skipButton.snp.makeConstraints { make in
+            // Check to see if iPhone has a notch
+            let bottom = UIApplication.shared.keyWindow?.safeAreaInsets.bottom ?? 0
+            if bottom > 10 {
```

Comment:
why do we check if its greater than `10`?

Reply:
Good catch! It should be bottom > 0

Reply:
I think it would be better to use `safeAreaInset/LayoutGuide/SomethingOrOther` to account for the notch. If you want me to provide more information about how to do this lmk in slack or here.